vimeo-networking/src/main/java/com/vimeo/networking/model/Credit.java

```diff
@@ -54,11 +54,19 @@ public Video getVideo() {
         return mVideo;
     }
 
+    public void setVideo(@Nullable Video video) {
```

Comment:
Do we want to make these `NonNull` instead? I know the field itself would still be nullable, but we are using these methods to update the fields, so it seems like we shouldn't allow someone to null out a non null field. What do you think?